docs/Reference.md

```diff
@@ -305,11 +305,11 @@ Specifying options allows to reuse a transformation in different settings.
 #### Java/Scala
 You can use Spark Dataset API in Java/Scala to define custom transformations. 
```

Comment:
I have never seen an example of an Extension of a CustomTransformer working with Datasets, only DataFrames. Would be interesting for some of our collegues.

Reply:
I'll change that to "Spark DataFrame API", but we could create a Transformer working with Dataset in the future. I'll create feature request for that.